libraries/support/azure/service.rb

```diff
@@ -28,6 +28,20 @@ def for_subscription(subscription_id, override: false)
       set_reader(:subscription_id, subscription_id, override)
     end
 
+    def structify(data)
```

Comment:
It only occurs to me with the distance of time that maybe I should've added a little documenting comment to this method. What do you think?